### PR TITLE
Version bump react-native to 0.55.4

### PR DESCRIPTION
**Pull request by tuncaulubilge**
React Context does not work with our current react-native version, so we had to bump it.

React-native bumped to 55.4

### CODE REVIEW

File: packages/article-flag/src/utils/beautify.android.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-// apply transformations to add uppercase and letter spacing.
-// letterSpacing CSS prop does not work on android:
-// https://github.com/facebook/react-native/pull/13199
-export default title =>
-  title
-    .toLowerCase()
-    .split("")
-    .join("\u200A");
```
Comment (colincclark) [inline packages/article-flag/src/utils/beautify.android.js]: 💯

File: packages/video-label/src/beautify-title.android.js [hunk 2]
```diff
@@ -1,9 +0,0 @@
-// apply transformations to add uppercase and letter spacing.
```
Comment (colincclark) [inline packages/video-label/src/beautify-title.android.js]: please amend the README which references this issue

Comment (tuncaulubilge) [reply packages/video-label/src/beautify-title.android.js]: Done :+1:

File: packages/utils/src/strings.js [hunk 3]
```diff
@@ -1,15 +1,5 @@
 /* eslint-disable import/prefer-default-export */
```
Comment (colincclark) [inline packages/utils/src/strings.js]: this is unrequired now I think?

Comment (tuncaulubilge) [reply packages/utils/src/strings.js]: Removed

**Comment by tuncaulubilge** [comment]
I couldn't find anything on react-native release notes about View to Component change. Compared a few snapshots, and seems like only the shallow rendered ones have changed to Component.

**Pull request merged**